fineract-savings/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountTransaction.java

```diff
@@ -127,10 +126,6 @@ public final class SavingsAccountTransaction extends AbstractAuditableWithUTCDat
     @Column(name = "reason_for_block", nullable = true)
     private String reasonForBlock;
 
-    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
I guess this is to reduce cross-dependencies to other domains? We should have more of these refactorings.